pages/signup/index.tsx

```diff
@@ -149,102 +150,104 @@ const SignUpPage = ({ t }: PageProps): React.ReactElement => {
       });
   };
 
+  console.log('signup', error);
+
+  layoutVar({ title: 'sign in', error });
```

Comment:
wouldn't line 148 be a good place to set the error? And a useEffect with an empty dependence array to set the null error in the beginning?

Reply:
this is being done in `handleSubmit` and `useEffect` now.